frontend/src/contexts/AuthContext.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { createContext, useState, useContext, useEffect } from 'react';
+import api from '../utils/api';
+
+const AuthContext = createContext(null);
+
+const useAuth = () => useContext(AuthContext);
+
+export const AuthProvider = ({ children }) => {
+  const [user, setUser] = useState(() => {
+    const savedUser = localStorage.getItem('user');
```

Comment:
Users need to be stored in the DB as well 
Again, use Prisma to interact with the DB

Reply:
Mb, if you are doing it already